lib/Dialect/ESI/ESIPasses.cpp

```diff
@@ -721,7 +721,7 @@ void ESIPortsPass::updateInstance(HWModuleExternOp mod, InstanceOp inst) {
     // Build a gasket by instantiating an interface, connecting one end to an
     // `esi.unwrap.iface` and the other end to the instance.
     auto ifaceInst =
-        instBuilder.create<InterfaceInstanceOp>(iface.getInterfaceType());
+        instBuilder.create<InterfaceInstanceOp>(iface.getInterfaceType(), None);
```

Comment:
Can this be defaulted in the builder?

Reply:
Yeah, a short custom builder can do this. Will add and land. Good thought. :+1: